### PR TITLE
Users assigning part of the Role management feature is implemented to the B2B sample application

### PR DESCRIPTION
**Pull request by Achintha444**
## Purpose
Initial phase of the Role management feature is implemented to the B2B sample application.
- Now a user can assign users for a role directly from the sample application

New role management interface
Screenshot 2022-11-11 at 18 15 27

### CODE REVIEW

File: b2b-sample/guardio-insurance-sample-application/components/sections/data/sideNav.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                },
+                {
+                    "title" : "Identity Providers",
+                    "eventKey" : "2-3",
+                    "type": "item"
+                }
+            ]
+        }
+    ]
+}
```
Comment (thivi) [inline b2b-sample/guardio-insurance-sample-application/components/sections/data/sideNav.json:34]: Add a new line

**Review by thivi — APPROVED** [review]

**Review by thivi — APPROVED** [review]

File: b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/general.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    const validate = values => {
+        let errors = {};
+
+        errors = nameValidate(values.name, errors);
+
+        return errors;
+    };
+
+    const onDataSubmit = (response, form) => {
+        if (response) {
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/general.js:62]: make the code consistent, by leaving a new line after the declaration. Check other places too.

File: b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/role/decodePatchRole.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                "op": patchMethod,
+                "path": path,
+                "value": value
+            }
+        ]
+    }
+}
+
+function getRemoveBody(patchMethod, path, value) {
+    return {
```
Comment (ThaminduR) [inline b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/role/decodePatchRole.js:37]: Keep a newline after the function declaration to make the code consistent. Check other places as well.

Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/role/decodePatchRole.js:37]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/83fabfed8c8201a0b1da8e023dbca5eb6494c56b

File: b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/general.js (hunk 2 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/general.js:62]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/83fabfed8c8201a0b1da8e023dbca5eb6494c56b

**Pull request merged**